t/python/t0005-kvs.py

```diff
@@ -378,7 +385,7 @@ def test_45_commit_fail(self):
 
     # just testing that passing flags work, these are pitiful KVS
     # changes and the flags don't mean much
-    def test_46_commit_flags(self):
+    def test_48_commit_flags(self):
```

Comment:
Er do we really need to renumber all these tests?  How about just adding an "a", "b", "c" to the test inserted in the middle, or drop the numbers in a separate commit?

(Other `unittest` users don't seem to be numbered so I assume it's not a necessary convention)

Reply:
I added the numbers a few PRs ago.  Basically the kvs tests grew enough in size that it was hard to debug when I was adding new tests.  Since the tests get run in lexicographical order, the test output was sort of all jumbled around in the output.  Maybe the right thing to do would be to split the tests up in the future so the file is a more manageable size, like many of the other tests.

Reply:
Oh ok.  Well, making the new tests 14a, 14b might work then?  Not a big deal if you feel strongly that you want it this way.

Reply:
I stopped numbering unittest tests because of this. The backtrace is usually enough to debug the failing tests

Reply:
The re-numbering was annoying ... ok you've convinced me... lets clean up the tests! (I'll do a follow up PR, since this one is already approved)